x-pack/plugins/maps/public/routes/map_page/map_app/map_app.tsx

```diff
@@ -264,14 +325,16 @@ export class MapApp extends React.Component<Props, State> {
     });
   };
 
-  // mapRefreshConfig: MapRefreshConfig
-  _onRefreshConfigChange(mapRefreshConfig: MapRefreshConfig) {
-    this.props.setRefreshConfig(mapRefreshConfig);
+  _onRefreshConfigChange({ isPaused, interval }: MapRefreshConfig) {
+    this.setState({
+      isRefreshPaused: isPaused,
+      refreshInterval: interval,
+    });
     updateGlobalState(
       {
         refreshInterval: {
-          pause: mapRefreshConfig.isPaused,
-          value: mapRefreshConfig.interval,
+          pause: isPaused,
+          value: interval,
```

Comment:
I understand these values were just updated, but following this update path, if I'm understanding correctly it looks like we're updating global state from global state. I wonder if we should short-circuit this?

Reply:
updating global state from global state is no longer occurring since calling of _onRefreshConfigChange has been reverted. Lets tackle this issue in a separate change.